pyproject.toml

```diff
@@ -25,7 +25,7 @@ pytest-celery = "^0.0.0"
 flask-apscheduler = "^1.12.4"
 
 [tool.pylint.messages_control]
-disable = "E1101, E0611, W0703, C0301, W0223, W0613, R0903, W0621"
+disable = "E1101, E0611, E0401, W0703, C0301, W0223, W0613, R0903, W0621"
```

Comment:
Why we added this error in ignore errors list? Can add what is this error and where did it occur. Let's take a look at all these error codes to see if we can correct those!